### PR TITLE
Automated cherry pick of #4749: Updated styles and table rows #4752: Add Calico Cloud #4933: Add correct path for cloud image

### PR DESCRIPTION
**Pull request by doublek**
Cherry pick of #4749 #4752 #4933 on release-v3.20.

#4749: Updated styles and table rows
#4752: Add Calico Cloud
#4933: Add correct path for cloud image

### CODE REVIEW

**Review by bmckercher123 — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
